clock/lib/clock.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Clock do
+  use Observable
+
+  @clock_speed 50
+
```

Comment:
I hate to diss an entire module, but would `:timer.send_interval` work here?

Reply:
Where do you think this code should live?  Still in a clock module or just in some sort of main module?